docs/user/codes/openmm.md

```diff
@@ -0,0 +1,639 @@
+    name="test_production",
+    makers=[
+        EnergyMinimizationMaker(
+            platform_name="CUDA",
+            platform_properties={"DeviceIndex": "0"},
+        ),
+        NPTMaker(),
+        OpenMMFlowMaker.anneal_flow(),
+        NVTMaker(),
+    ],
+)
+```
+
+Some systems (notably perlmutter) have multiple GPUs available on a
+single node. To fully leverage the compute, you'll need to distribute
+4 simulations across the 4 GPUs. A simple way to do this is with MPI.
+
+First you'll need to install mpi4py.
+
+```bash
+>>> conda install mpi4py
+```
+
+Then you can modify and run the following script to distribute the work across the GPUs.
+
+
+```python
+# other imports
+
+from mpi4py import MPI
+
+comm = MPI.COMM_WORLD
+rank = comm.Get_rank()
+
+list_of_mol_spec_lists = []
+# logic to add four mol_spec_lists to list_of_mol_spec_lists
+
+
+flows = []
+for i in range(4):
+    device_index = i
+    mol_specs = list_of_mol_spec_lists[i]
+
+    setup = generate_interchange(mol_specs, 1.0)
+
+    production_maker = OpenMMFlowMaker(
+        name="test_production",
+        makers=[
+            EnergyMinimizationMaker(
+                platform_name="CUDA",
+                platform_properties={"DeviceIndex": str(device_index)},
+            ),
+            NPTMaker(),
+            OpenMMFlowMaker.anneal_flow(),
+            NVTMaker(),
+        ],
+    )
+
+    production_flow = production_maker.make(
+        setup.output.interchange,
+        prev_task=setup.output,
+        output_dir=f"/pscratch/sd/o/oac/openmm_runs/{i}",
+    )
+    flows.append(Flow([setup, production_flow]))
+
+# this script will run four times, each with a different rank, thus distributing the work across the four GPUs.
+run_locally(flows[rank], ensure_success=True)
+```
+</details>
+
+# Analysis with Emmet
+
+For now, you'll need to make sure you have a particular emmet branch installed.
+Later the builders will be integrated into `main`.
+
+```bash
+pip install git+https://github.com/orionarcher/emmet.git@md_builders
+```
+
+### Analyzing Local Data
+
+<details>
+<summary>Learn to analyze your data without a database</summary>
+
+Emmet will give us a solid head start on analyzing our data even without touching
+a database. Below, we use emmet to create a [MDAnalysis Universe](https://docs.mdanalysis.org/stable/documentation_pages/core/universe.html#module-MDAnalysis.core.universe)
+and a [SolvationAnalysis Solute](https://solvation-analysis.readthedocs.io/en/latest/api/solute.html).
+From here, we can do all sorts of very cool analysis, but that's beyond the
+scope of this tutorial. Consult the tutorials in SolvationAnalysis and MDAnalysis
+for more information.
+
+```python
+from atomate2.classical_md.core import ClassicalMDTaskDocument
+from emmet.builders.classical_md.utils import create_universe, create_solute
+from openff.interchange import Interchange
+
+ec_emc_taskdoc = ClassicalMDTaskDocument.parse_file("tutorial_system/taskdoc.json")
```

Comment:
Better to just get the task doc from the database rather than relying on folders.